Test/benchmarks/sequence-race/SequenceRace.dfy

```diff
@@ -0,0 +1,45 @@
+
+// Ensure trying to use an unsupported compilation target results in a clean error message.
+// RUN: %exits-with 3 %baredafny translate cs %args "%s" --plugin:DafnyBenchmarkingPlugin.dll > "%t"
+// RUN: %diff "%s.expect" "%t"
+
+// RUN: %baredafny translate java %args "%s" --plugin:DafnyBenchmarkingPlugin.dll
+// RUN: mkdir -p %S/java/src/jmh
+// RUN: cp -r %S/SequenceRace-java %S/java/src/jmh/java
+
+// Note the intentional ">" as opposed to ">>", so we can check just the benchmark run output.
+// RUN: %S/java/gradlew jmh -p %S/java > "%t"
+// RUN: %OutputCheck --file-to-check "%t" "%s"
+// We manually verify the benchmark fails (since the bug isn't fixed yet), because unfortunatley
+// we can't trust the JMH gradle plugin to: https://github.com/melix/jmh-gradle-plugin/issues/255
+// CHECK: <failure>
```

Comment:
Where does this `<failure>` message come from? JMH? And what exactly does it indicate?

Reply:
The actual output looks something like this:

```
...
Iteration   1: 1750958880.522 ops/s
Iteration   2: 1694825651.837 ops/s
Iteration   3: 1730645831.331 ops/s
Iteration   4: 1661990800.884 ops/s
Iteration   5: 1623016118.042 ops/s
Iteration   6: <failure>

java.lang.NullPointerException
        at java.base/java.util.ArrayDeque.addFirst(ArrayDeque.java:286)
        at java.base/java.util.ArrayDeque.push(ArrayDeque.java:579)
        at dafny.ConcatDafnySequence.computeElements(DafnySequence.java:845)
        at dafny.ConcatDafnySequence.force(DafnySequence.java:817)
        at dafny.LazyDafnySequence.select(DafnySequence.java:752)
        at _System.SequenceRace.LazyRace(SequenceRace.java:27)
        at _System.jmh_generated.SequenceRace_LazyRace_jmhTest.LazyRace_Throughput(SequenceRace_LazyRace_jmhTest.java:97)
        at jdk.internal.reflect.GeneratedMethodAccessor1.invoke(Unknown Source)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:568)
        at org.openjdk.jmh.runner.BenchmarkHandler$BenchmarkTask.call(BenchmarkHandler.java:475)
        at org.openjdk.jmh.runner.BenchmarkHandler$BenchmarkTask.call(BenchmarkHandler.java:458)
        at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
        at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
        at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
        at java.base/java.lang.Thread.run(Thread.java:833)
 ...
```

`<failure>` was the best indicator I could find that an exception occurred, without being overly precise about which one (since for all I know future bugs could throw something different)